app/reducers/index.js

```diff
@@ -9,6 +9,27 @@ const defaultState = {
   currentGoal: {},
 };
 
+export const saveRating = (state, time, id) => {
+  const newRating = {
+    score: state.currentGoal.newRating.score,
+    id: id,
+    time: time,
+    comment: state.currentGoal.newRating.comment,
+  };
+  const currentGoal = {
+    ...state.currentGoal,
+    ratings: [newRating].concat((state.currentGoal.ratings || [])),
```

Comment:
I think we need to increment the update count here.. I would make sure that you have a single function that is responsible for changing a goal. Feel free to talk to me about this!